### PR TITLE
[doc] PR for Add docs for universe level kubernetes overrides

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6514

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 1]
```diff
@@ -88,7 +93,19 @@ Complete the **G-Flags** section as follows:
...
+
+  ![img](/images/yb-platform/kubernetes-config66.png)
+
+- Using the sample format, specify the universe-level overrides for YB-Master and YB-TServer, as well as availability zone overrides.
+
+- Select **Force Apply** if you want to override any previous overrides.
+
+- Click **Validate & Save**. 
+
+  If there are any errors in your overrides definitions, a detailed error message is displayed. You can correct the errors and try to save again.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:108]: The force apply is used to override any validation errors. So if there are validation errors but you want to go ahead and save, you would use this checkbox

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 2]
```diff
@@ -88,7 +93,19 @@ Complete the **G-Flags** section as follows:
...
   - [YB T-Server Configuration Flags](/preview/reference/configuration/yb-tserver/#configuration-flags)
 
-Accept default values for all of the remaining fields.
+Optionally, use the **Helm Overrides** section as follows:
+
+- Click **Add Kubernetes Overrides** to open the **Kubernetes Overrides** dialog shown in the following illustration:
+
+  ![img](/images/yb-platform/kubernetes-config66.png)
+
+- Using the sample format, specify the universe-level overrides for YB-Master and YB-TServer, as well as availability zone overrides.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:102]: We should include the following details

1. Overrides are specified in yaml format, for example
```
master:
  podLabels:
    service-type: 'database'
```
2. Overrides need to be valid helm values from the [yugabyte helm chart](https://github.com/yugabyte/charts/blob/master/stable/yugabyte/values.yaml)
3. Availability zone level overrides only apply to pods that are deployed int that specific availability zone. To specify overrides for availability zone us-west-2a, one would specify yaml of the form

```
us-west-2a:
  master:
    podLabels:
       service-type: 'database'
```
4. When conflicting overrides are specified, the following order of precedence is used : universe availability zone level overrides, universe-level overrides, provider overrides

**Comment by iSignal** [comment]
Thanks for writing this up Liza!

We also support a day 2 operation called "Edit Kubernetes Overrides", which allows users to update these overrides at a later point after creation. Here are some screenshots for this feature

image

image

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:108]: Fixed

**Comment by lizayugabyte** [comment]
@athulsn @govardhanjalla Could you take a look at this doc PR, please?

**Comment by athulsn** [comment]
not able to preview, Can you please share the preview link ? @lizayugabyte

**Comment by lizayugabyte** [comment]
@athulsn I'm sorry, I don't know what happened to the preview build. I refreshed the commit. Here's the preview link: https://deploy-preview-15208--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes/

**Comment by athulsn** [comment]
Thanks @lizayugabyte !

File: docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md [hunk 3]
```diff
@@ -43,42 +43,47 @@ If no universes have been created yet, the **Dashboard** does not display any.
...
+- Use the **Enable Client-to-Node TLS** field to specify whether or not to enable encryption-in-transit for communication between clients and the database servers. This setting is enabled by default.
+- Use the **Enable Encryption at Rest** field to specify whether or not to enable encryption for data stored on the tablet servers. This setting is disabled by default.
 
 Complete the **Advanced** section as follows:
 
 - In the **DB Version** field, specify the YugabyteDB version. The default is either the same as the YugabyteDB Anywhere version or the latest YugabyteDB version available for YugabyteDB Anywhere.
+- Use the **Enable IPV6** field to specify whether or not you want to use IPV6 networking for connections between database servers. This setting is disabled by default.
+- Use the **Enable Public Network Access** field to specify whether or not to assign a load balancer or nodeport for connecting to the database endpoints over the internet. This setting is disabled by default.
+- Use the **Enable Systemd Services** field to specify whether or not systemd services are allowed to run. This setting is enabled by default.
```
Comment (athulsn) [inline docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md:80]: I guess **Enable Systemd Services**  is not available for kubernetes universes  @iSignal @lizayugabyte

**Review by athulsn — COMMENTED** [review]
Rest looks good

**Review by athulsn — APPROVED** [review]

**Comment by lizayugabyte** [comment]
@iSignal I implemented review comments. Could you, please, take a look and sign off on the doc?

**Pull request merged**